ote/src/clj/ote/tasks/pre_notices.clj

```diff
@@ -130,22 +130,18 @@
              (log/warn "Error while sending a notification" e))))))))
 
 
-(defrecord PreNoticesTasks [at email-config interval]
+(defrecord PreNoticesTasks [email-config]
   component/Lifecycle
   (start [{db :db :as this}]
     (assoc this
-      ::stop-tasks [(chime-at (drop 1 (periodic-seq at (t/seconds
-                                                         (hours->seconds (or interval
```

Comment:
Why did you remove the usage of interval option? It is much easier to change task intervals from config.edn than in code, no new build is needed.

Reply:
I think we don't need it, it is even easier to call the timed function from the RPEL for testing purposes.

The `daily-at` gives a sequence of daily times (without an interval).

Reply:
Ok, could you also remove the interval settings from config.end and aws config.edn templates?